app/src/main/java/uk/gov/android/securestore/crypto/limitedmanager/LimitedCryptoManager.kt

```diff
@@ -0,0 +1,49 @@
+package uk.gov.android.securestore.crypto.limitedmanager
+
+/**
+ * Interface to handle encryption and decryption of [String] data
+ */
+interface LimitedCryptoManager {
```

Comment:
Is there a specific reason we are using callbacks here? Its not a paradigm that we are using in the rest of the projects.